app/src/main/cpp/skyline/services/nvdrv/devices/nvhost_ctrl.cpp

```diff
@@ -1,81 +1,109 @@
 // SPDX-License-Identifier: MPL-2.0
 // Copyright © 2020 Skyline Team and Contributors (https://github.com/skyline-emu/)
+// Copyright © 2019-2020 Ryujinx Team and Contributors
```

Comment:
Copyright always needs to be done on a source level, and and I'm fairly sure there can be two of them, this can be seen in many Linux files iirc

Reply:
https://github.com/torvalds/linux/blob/fcadab740480e0e0e9fa9bd272acd409884d431a/ipc/compat.c

Reply:
> Copyright always needs to be done on a source level  

As far as I know, it can be done at either level but with the per-file scope of MPL I suppose it is better to follow that. We will try our best at annotating source with Ryujinx's copyright wherever derivative work has been used and also add the appropriate binary notices soon enough (It's a part of the UI Rework TODO).

Unrelated, but, this did remind me that we need to update the copyright year in all Skyline source files.